frontend/src/components/pages/PlayPage.js

```diff
@@ -22,11 +22,13 @@ export default function PlayPage() {
       <body>
```

Comment:
´<body>` is already defined in `index.html`. Only one body element allowed per page!